var/spack/repos/builtin/packages/omega-h/package.py

```diff
@@ -34,18 +34,31 @@ class OmegaH(CMakePackage):
 
     homepage = "https://github.com/ibaned/omega_h"
     url      = "https://github.com/ibaned/omega_h/archive/v9.13.4.tar.gz"
+    git      = "https://github.com/ibaned/omega_h.git"
 
-    version('9.13.4', '035f9986ec07ad97ae0aa1e171872307')
+    version('master', branch='master')
+    version('9.13.14', sha256='f617dfd024c9cc323e56800ca23df3386bfa37e1b9bd378847d1f5d32d2b8e5d')
+    version('9.13.13', sha256='753702edf4bda9ae57ea21f09ca071e341604a468d8c86468c9aebba049f581c')
+    version('9.13.12', sha256='b37872c6059d5405f971098cce7bab3f74926d99b5f9dc8f8270467a85ed41d7')
+    version('9.13.11', sha256='992ba790cabc07f2c913c3f1e7ab28a46a76084567c39f69b77e4757e05b8720')
 
     variant('shared', default=True, description='Build shared libraries')
     variant('mpi', default=True, description='Activates MPI support')
     variant('zlib', default=True, description='Activates ZLib support')
+    variant('trilinos', default=True, description='Use Teuchos and Kokkos')
+    variant('build_type', default='')
+    variant('gmodel', default=True, description='Gmsh model generation library')
+    variant('throw', default=False, description='Errors throw exceptions instead of abort')
+    variant('examples', default=False, description='Compile examples')
 
+    depends_on('gmodel', when='+gmodel')
+    depends_on('gmsh', when='+examples', type='build')
```

Comment:
just to be sure: gmsh is not linked?

Reply:
Indeed gmsh is not linked. It is used at build time to create .msh from .geo

Reply:
it's actually not even really a requirement for build - only for running the tests, which is in the CMake files. Maybe there should be a specific target for this in the zee cmake.